snuba/datasets/storages/querylog.py

```diff
@@ -30,29 +31,57 @@
         ("timestamp", DateTime()),
         ("duration_ms", UInt(32)),
         ("status", LowCardinality(String())),
+        # clickhouse_queries Nested columns.
```

Comment:
When James tried to create the new columns (they did not have a default) we noticed that clickhouse would refuse all INSERTS that did not include the newly added columns (which were not there as expected because the consumer changes happens after the schema change).
The issue clickhouse raised was that, when the new columns were missing the whole clickhouse_queries nested column would not have the same numebr of rows on all fields.

So we needed to add a default for the new columns. Which are arrays of arrays, and the default has to be an array of the same size of clickhouse_queries.sql. Now such schema is not achievable with the `Nested` construct, where we cannot set a default for an entire column inside a nested field.

This means that adding columns with an alter in a nested column generates a schema that cannot be represented with a CREATE table statement. 

If we want to solve this we need to rethink the way our `Nested` api works to be able to match a schema where some columns are added after the creation of the table.